strongbox-cron/strongbox-cron-api/src/main/java/org/carlspring/strongbox/cron/jobs/AbstractCronJobsRegistry.java

```diff
@@ -0,0 +1,37 @@
+package org.carlspring.strongbox.cron.jobs;
+
+import java.lang.reflect.Modifier;
+import java.util.Set;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+
+import com.google.common.collect.ImmutableSet;
+import org.reflections.Reflections;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+@Component
+class AbstractCronJobsRegistry
+        implements Supplier<Set<Class<? extends AbstractCronJob>>>
+{
+
+    private final Set<Class<? extends AbstractCronJob>> cronJobs;
+
+    AbstractCronJobsRegistry()
+    {
+        cronJobs = ImmutableSet.copyOf(new Reflections("org.carlspring.strongbox")
```

Comment:
we can probably restrict the package to `org.carlspring.strongbox.cron.jobs`, it will work much faster in this case.

Reply:
yeah, will check that, thanks 👍